src/index.html

```diff
@@ -7,7 +7,47 @@
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body >
+    <main class="card" data-qa="card">
+
+      <img
+      src="../src/images/imac.jpeg"
+      alt="main picture"
+      class="card__image">
+
+
+      <section class="card__discription">
+        APPLE A1419 iMac 27" Retina<br>5K Monoblock (MNED2UA/A)
+      </section>
+
+      <section class="card__prodact">
+        Product code: 195434
+      </section>
+
+      <section class="card__review">
+        <div class="card__review-stars">
```

Comment:
Add the modifier classes
```suggestion
        <div class="stars stars--4">
```

Reply:
Still not fixed